kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/RecordSerDe.java

```diff
@@ -10,9 +12,17 @@
 public interface RecordSerDe {
 
   @Value
+  @Builder
   class DeserializedKeyValue {
     @Nullable String key;
     @Nullable String value;
+    @Nullable MessageFormat keyFormat;
+    @Nullable MessageFormat valueFormat;
+    @Nullable Integer keySize;
+    @Nullable Integer valueSize;
+    @Nullable String keySchemaId;
+    @Nullable String valueSchemaId;
+    @Nullable Integer headersSize;
```

Comment:
**Minor:** why we decided to add keySize, valueSize, headersSize in **Deserialized**KeyValue ? It looks like it can be calculated with out of deserialization logic..

Reply:
I agree. Will move this logic out of deserialization.

Reply:
Fixed, please review